components/AboutSectionInfoCard/AboutInfoCard.tsx

```diff
@@ -21,7 +21,7 @@ export default function AboutInfoCard({ title, bodytext }: AboutInfoCardProps) {
       <Card className="h-100 bg-white">
         <StyledCardHeader
           forwardedAs="h3"
-          className="text-center align-self-center bg-info text-white rounded-0 border-0 mb-n3 overflow-visible"
+          className="text-center align-self-center bg-warning text-white rounded-0 border-0 mb-n3 overflow-visible"
```

Comment:
because of the limited number of bootstrap color variables, I moved orange from bg-info to bg-warning so that I can use bg-info elsewhere

also moving orange to bg-warning more closely mimics the bootstrap defaults

bg-info had only been used in 2 places and visually the user experience is the same